SocialFilter/sections/specific.txt

```diff
@@ -63,6 +63,7 @@ allets.ru,annahelp.ru,newslenta.com,kogdavyydet.com,u74.ru,wifika.ru,mater.com.u
 !
+||push.zhanzhang.baidu.com^
```

Comment:
Shouldn't it be added to social_trackers? @Alex-302 please take a look.

Reply:
As I see, this is analytics. Please look at script's content.

Reply:
I had seen it and thougth the same, but wasn't sure for 100% that's why I mentioned you. Ok.

Reply:
@piquark6046 please move it to AdGuard Tracking Protection, it looks like a tracker and I agree with it https://github.com/AdguardTeam/AdguardFilters/pull/140316#discussion_r1072192956.